src/settings/JobProfiles/JobProfileDetails/JobProfileDetails.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { noop } from 'lodash';
+
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Col,
+  ExpandAllButton,
+  Headline,
+  KeyValue,
+  Layer,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+import {
+  FullScreenView,
+  Preloader,
+  SettingsLabel,
+} from '@folio/stripes-data-transfer-components';
+
+const JobProfileDetails = props => {
+  const {
+    jobProfile,
+    mappingProfile,
+    loading,
+    stripes,
+    onCancel,
+  } = props;
+
+  return (
+    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
+      {contentLabel => (
+        <Layer
+          isOpen
+          contentLabel={contentLabel}
+        >
+          <FullScreenView
+            id="job-profile-details"
+            paneTitle={jobProfile?.name && <SettingsLabel iconKey="jobProfiles">{jobProfile.name}</SettingsLabel>}
+            onCancel={onCancel}
+          >
+            {loading
+              ? <Preloader />
+              : (
+                <AccordionStatus>
+                  <Row>
+                    <Col xs={9}>
+                      <Headline
+                        data-test-headline
+                        size="x-large"
+                        tag="h2"
+                        margin="small"
+                      >
+                        {jobProfile.name}
+                      </Headline>
+                    </Col>
+                    <Col xs={3}>
+                      <Row end="xs">
+                        <Col xs>
+                          <ExpandAllButton />
```

Comment:
It seems, there are not accordions and the expand button on mockups. I wonder, do we really need it here?

Reply:
Good catch. Will remove.